package exceptionHandling;

public class ThrowKeyword {
	
	
	public static void main(String[] args) {
		
		
		System.out.println("Before Exception");
		
		throw new ArithmeticException("Testing Throw keyword");
	}

}
